spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'liking photos' do
+  before do
+  	@user = User.create email: 'natso@gmail.com', password: '12345678', password_confirmation: '12345678'
+    login_as @user
+    allow_any_instance_of(Paperclip::Attachment).to receive(:url).and_return("/system/photos/images/000/000/001/original/gramophone.png")
+    @photo = Photo.create description:'nice', image_file_name:'spec/fixtures/files/gramophone.png', user_id: @user.id
```

Comment:
Line is too long. [117/80]<br>Inconsistent indentation detected.<br>Space missing after colon.